meson.build

```diff
@@ -0,0 +1,47 @@
+project('hidapi', 'c', version: files('VERSION'),meson_version: '>=0.60')
+
+iconv_dep = dependency('iconv', required: get_option('libusb'))
+libusb_dep = dependency('libusb-1.0', required: get_option('libusb'))
+udev_dep = dependency('libudev', required: get_option('hidraw'))
+
+incdirs = include_directories('hidapi')
+
+install_headers('hidapi/hidapi.h', subdir: 'hidapi')
+
+pconf = import('pkgconfig')
+
+if libusb_dep.found() and iconv_dep.found()
+  libhidapi = library('hidapi-libusb',
+    'libusb/hid.c',
+    dependencies: [ iconv_dep, libusb_dep ],
+    include_directories: incdirs,
+    version: files('VERSION'),
```

Comment:
This works for project, specifically.

For libraries just use meson.project_version()...